edweiss-app/app/(app)/courses/[id]/archive.tsx

```diff
@@ -68,7 +68,7 @@ const ArchiveScreen: ApplicationRoute = () => {
 
                 {assignments.length > 0 ?
                     assignments.map((assignment) => (
-                        <AssignmentDisplay item={assignment.data} id={assignment.id} courseID={courseId} index={assignments.indexOf(assignment)} isSwipeable={false} key={assignment.data.name} />
+                        <AssignmentDisplay item={assignment.data} id={assignment.id} courseID={courseId} index={assignments.indexOf(assignment)} key={assignment.data.name} />
                     ))
```

Comment:
You can find index directly in map component 

`    assignments.map((assignment, index) => (`
it's faster 
for the key i think it's not a good idee to use the name because multiple assignment can have the same name. you can use the id

Reply:
Yes you are right I don't know why this has been on the main branch all this time